XMonad/Layout/Inspect.hs

```diff
@@ -8,15 +8,17 @@
 
 -- |
 -- Module      :  XMonad.Layout.Inspect
+-- Description :  Inspect layout data.
```

Comment:
Perhaps `s/data/state` (here and in other places)?  Data sounds very static, but the main point seems to be that we want access to runtime information